src/routes/sitemaps.ts

```diff
@@ -25,4 +34,49 @@ export default async function (fastify: FastifyInstance): Promise<void> {
       .header('cache-control', 'public, max-age=14400')
       .send(stream);
   });
+
+  fastify.get('/robots.txt', async (req, res) => {
+    const con = await createOrGetConnection();
+    const bots = con
+      .createQueryBuilder()
+      .select('username')
+      .from(BotUser, 'b')
```

Comment:
How is `bot_user` table filled?

Reply:
Retool app currently that is manually reviewed by me. Later I'll do something more automatic